src/main/scala/org/broadinstitute/dsde/workbench/leonardo/model/LeonardoModel.scala

```diff
@@ -82,23 +84,23 @@ object ClusterStatus extends Enumeration {
 
 
 object Cluster {
-  def create(clusterRequest: ClusterRequest, clusterName: ClusterName, googleProject: GoogleProject, googleId: UUID, operationName: OperationName, serviceAccount: WorkbenchEmail, clusterDefaultsConfig: ClusterDefaultsConfig): Cluster = {
+  def create(clusterRequest: ClusterRequest, clusterName: ClusterName, googleProject: GoogleProject, googleId: UUID, operationName: OperationName, serviceAccount: WorkbenchEmail, clusterDefaultsConfig: ClusterDefaultsConfig, clusterStatus:ClusterStatus): Cluster = {
     Cluster(
-        clusterName = clusterName,
-        googleId = googleId,
-        googleProject = googleProject,
-        googleServiceAccount = serviceAccount,
-        googleBucket = clusterRequest.bucketPath,
-        machineConfig = MachineConfig(clusterRequest.machineConfig, clusterDefaultsConfig),
-        clusterUrl = getClusterUrl(googleProject, clusterName),
-        operationName = operationName,
-        status = ClusterStatus.Creating,
-        hostIp = None,
-        createdDate = Instant.now(),
-        destroyedDate = None,
-        labels = clusterRequest.labels,
-        jupyterExtensionUri = clusterRequest.jupyterExtensionUri
-      )
+      clusterName = clusterName,
+      googleId = googleId,
+      googleProject = googleProject,
+      googleServiceAccount = serviceAccount,
+      googleBucket = clusterRequest.bucketPath,
+      machineConfig = MachineConfig(clusterRequest.machineConfig, clusterDefaultsConfig),
+      clusterUrl = getClusterUrl(googleProject, clusterName),
+      operationName = operationName,
+      status = clusterStatus,
```

Comment:
Hm.  I don't like this from a semantic standpoint.  I want `create()` to always produce a `Creating` cluster.  It's easy to change the status in tests post-creation